provider/resources.go

```diff
@@ -676,6 +677,7 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_cloudformation_stack":              {Tok: awsResource(cloudformationMod, "Stack")},
 			"aws_cloudformation_stack_set":          {Tok: awsResource(cloudformationMod, "StackSet")},
 			"aws_cloudformation_stack_set_instance": {Tok: awsResource(cloudformationMod, "StackSetInstance")},
+			"aws_cloudformation_type":               {Tok: awsResource(cloudformationMod, "CFType")},
```

Comment:
Does `Type` conflict with some built-ins? Shall we make it `CloudFormationType` then?

Reply:
Yeah it was conflicting in Go SDK... If you think CloudFormationType is better then I will go with that